kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -1286,17 +1286,6 @@ protected void handleOffsetCommitRequest(KafkaHeaderAndRequest offsetCommit,
 
     protected void handleFetchRequest(KafkaHeaderAndRequest fetch,
                                       CompletableFuture<AbstractResponse> resultFuture) {
-        long startFetchingRequestNanos = MathUtils.nowInNano();
-        resultFuture.whenComplete((r, e) -> {
-            if (e != null) {
-                requestStats.getHandleFetchRequestStats().registerFailedEvent(
-                        MathUtils.elapsedNanos(startFetchingRequestNanos), TimeUnit.NANOSECONDS);
-            } else {
-                requestStats.getHandleFetchRequestStats().registerSuccessfulEvent(
-                        MathUtils.elapsedNanos(startFetchingRequestNanos), TimeUnit.NANOSECONDS);
-            }
-        });
```

Comment:
It's removed because https://github.com/streamnative/kop/pull/514 has replaced it with `REQUEST_LATENCY`, right? And the `handleFetchRequestStats` field of `RequestStats` should be removed as well. Also `HANDLE_PRODUCE_REQUEST` could be removed too.

In addition, I found the `testMetricsProvider` still passed. I found the metrics string contains `NaNkop_server_HANDLE_FETCH_REQUEST`. Should we improve the tests to detect some unused metrics related fields?

Reply:
Yes, #514 has introduced all apis latency metric, so we should remove the above one. Some field should also be removed, it will update the code.

The test still passed is due to the metric has been initialized, and just not update the value.